lapce-ui/src/title.rs

```diff
@@ -126,7 +148,15 @@ impl Widget<LapceWindowData> for Title {
         _data: &LapceWindowData,
         _env: &Env,
     ) -> Size {
-        Size::new(bc.max().width, 28.0)
+        #[cfg(not(target_os = "windows"))]
+        {
+            Size::new(bc.max().width, 28.0)
+        }
+
+        #[cfg(target_os = "windows")]
+        {
+            Size::new(bc.max().width, 32.0)
+        }
```

Comment:
Changed titlebar height on Windows to match most other apps and improve readability slightly